<launch>

    <node name="pcloud_stich" pkg="visavis_vision" type="visavis_pcloud_stich" respawn="false" output="screen">
        <remap from="/in_pointcloud" to="/camera/depth_registered/points"/>
        <!-- <remap from="/stiched_pointcloud" to="/camera/depth_registered/points"/> -->
        
        <param name="voxel_size_stiching" value="0.1"/>
        <param name="voxel_size_input_cloud" value="0.1"/>
        <param name="max_cam_depth" value="3.0"/>
        <param name="output_reference_frame" value="world"/>
    </node>
</launch>